hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Tables/details/ReviewOrgUnitChangesDetailsTableRow.tsx

```diff
@@ -22,24 +22,27 @@ const useStyles = makeStyles(theme => ({
     },
     cellRejected: {
         '& > a': {
-            color: theme.palette.error.main,
+            color: `${theme.palette.error.main}! important`,
```

Comment:
Are you sure that this works?
Shouldn't the `!important` property be written like that?

```suggestion
            color: `${theme.palette.error.main} !important`,
```